src/databricks/labs/ucx/framework/crawlers.py

```diff
@@ -41,6 +41,27 @@ def _schema_for(cls, klass):
             fields.append(f"{f.name} {spark_type}{not_null}")
         return ", ".join(fields)
 
+    @classmethod
+    def _filter_none_rows(cls, rows):
+        results = []
+        nullable_fields = []
+
+        for field in dataclasses.fields(rows[0]):
+            if field.default is None:
+                nullable_fields.append(field.name)
+
+        for row in rows:
```

Comment:
Also add `if row is None: continue` - might be enough to fix the issue, but it's good to be more defensive